src/effects/NoiseReduction.cpp

```diff
@@ -694,6 +694,10 @@ bool EffectNoiseReduction::Process(EffectInstance &, EffectSettings &)
    this->ReplaceProcessedTracks(bGoodResult);
    return bGoodResult;
 }
+EffectUIClientInterface* EffectNoiseReduction::GetEffectUIClientInterface()
+{
+   return nullptr;
+}
```

Comment:
I don't understand why NoiseReduction required special treatment.  Can you explain?

This new function needs to be called only from RealtimeEffectStateUI, and Noise Reduction is not one of the realtime capable effects, so I don't understand how you can ever reach this override.

Reply:
As you know, I do not trust in "something is not called now" :-) The reason `GetEffectUIClientInterface` is there for symmetry. I had some reasoning why this was not done for `NyquistEffect`, but I don't remember it.